HW04/Lizaveta_Muromskaya/mentor.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require './human'
+# The class that is responsible for the mentor's actions
+class Mentor < Human
+  attr_reader :notifications
+  attr_accessor :students
```

Comment:
No need to use `accessor` here, `reader` should be enough and you can move it to the `private` section if you're not going to use this method outside of this class
```suggestion
  private 
  
  attr_reader :students
```